spec/api/v4/drugs_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "swagger_helper"
+
+describe "Drugs v4 API", swagger_doc: "v4/swagger.json" do
+  path "/drugs/sync" do
+    get "Syncs drug data from server to device." do
+      tags "Drug"
+      parameter name: "HTTP_X_FACILITY_ID", in: :header, type: :uuid
```

Comment:
Making the facility_id a part of this spec, since it'll become relevant when we switchover to using this API for all drug info. 
Haven't made it mandatory though, so requests without facility_id will still work for now. Maybe it should be mandatory anyway?

Reply:
I think non-mandatory should be fine, we seem to allow it for other resources too.